### PR TITLE
some api calls and UI

### PR DESCRIPTION
**Pull request by bhtibrewal**
working on UI screens along with some API calls

### CODE REVIEW

File: src/components/right_sidebar/RightSidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { getUsers } from "../../services";
+import "./right_sidebar.css";
+export const RightSidebar = () => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    getUsers({ setUsers });
+  }, []);
+  console.log(users);
```
Comment (yaswanthmyneni) [inline src/components/right_sidebar/RightSidebar.jsx:10]: changes: remove console.log

File: src/pages/user_profile/user_profile.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  gap: 1rem;
+}
+.stats-cards {
+  background-color: var(--light-text);
+  border-radius: 4px;
+  padding: 1rem;
+}
+.stats-card-icon {
+  border-radius: 1rem;
+  padding: 0.5rem;
```
Comment (yaswanthmyneni) [inline src/pages/user_profile/user_profile.css:13]: changes: we can use px for less than 16px

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
This is small PR, though I have some UI suggestions

### You can make create new post component as same width as post itself and can add gap between 2 posts
If you notice any popular Social Media like Facebook and Twitter, they also do that
Capture3
Capture4

File: src/services/posts/createPost.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios"
+
+export const createPost = async ({postData}) => {
+    try {
+        const res = await axios.post(`/api/posts`, {postData});
+    }
+    catch (e) {
+
```
Comment (yaswanthmyneni) [inline src/services/posts/createPost.js:8]: changes: not there any code inside catch block everywhere

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/services/posts/createPost.js (hunk 3 above)
Comment (bhtibrewal) [reply src/services/posts/createPost.js:8]: It is left purposely for future use

File: src/pages/user_profile/user_profile.css (hunk 2 above)
Comment (bhtibrewal) [reply src/pages/user_profile/user_profile.css:13]: Yes will do that, thanks for pointing out

File: src/components/right_sidebar/RightSidebar.jsx (hunk 1 above)
Comment (bhtibrewal) [reply src/components/right_sidebar/RightSidebar.jsx:10]: Yes missed it, will do

**Pull request merged**